packages/select-rich/src/LionSelectRich.js

```diff
@@ -388,6 +399,7 @@ export class LionSelectRich extends ScopedElementsMixin(
 
     this._listboxNode.addEventListener('active-changed', this.__onChildActiveChanged);
     this._listboxNode.addEventListener('model-value-changed', this.__proxyChildModelValueChanged);
+    this.addEventListener('keydown', handleKeydown);
```

Comment:
Could we move this to LionSelectRichInvoker? Because that's the actual `activeElement` when you tab into the rich select. Right now this "works" because LionSelectRich happens to be a parent, so the listener is triggered, but I think it would be better to move it to the invoker where the event happens.

Reply:
Yes it's working when moved in the invoker